plugin/src/test/java/io/jenkins/plugins/casc/vault/KV1WithUserTest.java

```diff
@@ -22,6 +22,8 @@
 
     @BeforeClass
     public static void configureVaultContainer() {
+        // Dont run on non-docker daemon nodes
+        org.junit.Assume.assumeTrue(VaultTestUtil.hasDockerDaemon());
```

Comment:
you could static import assumeTrue and VaultTestUtil the tests for some more readability :sweat_smile:

Reply:
Good point.
Done :+1: